core/templates/pages/release-coordinator-page/jobs-tab/jobs-tab.component.spec.ts

```diff
@@ -132,43 +122,43 @@ describe('Admin Jobs Tab Component', () => {
 
     fetchJobOutputAsync(jobId: string): FakeThen {
       let adminJobOutputUrl = urlInterpolationService.interpolateUrl(
-        AdminPageConstants.ADMIN_JOB_OUTPUT_URL_TEMPLATE, {
+        '/fetchJobOutputAsync/<job_id>', {
           jobId: jobId
         });
       return this.get(adminJobOutputUrl);
     }
 
-    getDataAsync(): FakeThen {
+    getJobsDataAsync(): FakeThen {
       return {
-        then: (callback: (adminDataObject: AdminPageData) => void) => {
-          callback(mockAdminPageData);
+        then: (callback: (adminDataObject: JobsData) => void) => {
+          callback(mockJobsData);
         }
       };
     }
 
     startNewJobAsync(jobType: string): FakeThen {
-      return this.post(AdminPageConstants.ADMIN_HANDLER_URL, {
+      return this.post('/startNewJobAsync', {
```

Comment:
Why not add this as a constant?

Reply:
Because this is a test file and these are mock/unused values. [The structure of this test file is weird!]

Reply:
I don't understand -- what do you mean by weird? Can we make it not weird?

Reply:
Vojtech: Expecting this mock function can have any value I've used a string constant.

Current test is using lots of logics and mocks the entire service which is not expected! Also, this a mock function which is using a mock this.post method so it doesn't need a specific URL.

I've created an issue at #12930 and will fix it in a seperate PR. Also, work with angular team to write better testing ideas in wiki page.